### DISCUSSION TITLE
Why ram-bundle can't use esbuild?

### DISCUSSION THREAD

**Discussion by Hao-yiwen** [microsoft/rnx-kit#3211] [Q&A] [answered]
<img width="972" alt="image" src="https://github.com/microsoft/rnx-kit/assets/42827938/b2e44325-b38d-4fc1-9eb5-62a069b188de">

**Comment by Hao-yiwen** [comment]
who can tell me why?This question always bothers me.

**Reply by tido64** [reply, marked as answer]
As mentioned in the screenshot you posted, Metro ignores the custom serializer. It does not get called at all. There are no hooks for the `ram-bundle` command provided by Metro. It is unlikely that we will ever support this as `ram-bundle` is no longer supported: https://github.com/facebook/react-native/commit/58b45e86a6fd78409dc358e27ae736c95fee9f39
